crates/swc_plugin_import/tests/fixtures/custom-name-tpl-camel/option.json

```diff
@@ -3,7 +3,7 @@
     "pluginImport": [
       {
         "libraryName": "foo",
-        "customName": "foo/__custom_es__/{{kebabCase member }}"
+        "customName": "foo/__custom_es__/{{ camelCase member }}"
```

Comment:
Thanks, but I don't think these cases are working anymore here. They are now one of builtin-swc-loader's feature, tests of which are located here `packages/rspack/tests/configCases/builtin-swc-loader/plugin-import`.

Reply:
OK, I plan to move all the `swc_plugin_import` test cases to `configCases/builtin-swc-loader/plugin-import-cases`